bika/lims/workflow/__init__.py

```diff
@@ -376,6 +376,13 @@ def getCurrentState(obj, stateflowid='review_state'):
     wf = getToolByName(obj, 'portal_workflow')
     return wf.getInfoFor(obj, stateflowid, '')
 
+def in_state(obj, states, stateflowid='review_state'):
+    """ Returns if the object passed matches with the states passed in
+    """
+    if not states:
+        return False
+    obj_state = getCurrentState(obj, stateflowid=stateflowid)
+    return obj_state in states
```

Comment:
Please note that `getCurrentState` returns an empty string as default (`return wf.getInfoFor(obj, stateflowid, '')`)

Reply:
Good catch. I take note, this change requires a new PR imo, because has nothing to do with `in_state` rather with `getCurrentState`.